scss/_nav.scss

```diff
@@ -225,7 +231,7 @@
   // Boosted mod
   > .tab-pane-with-nested-tab {
     display: none;
-    margin: $spacer * -1 calc(subtract($nav-tabs-link-padding-x, $nav-tabs-border-width) * -1); // stylelint-disable-line function-disallowed-list
+    margin: calc(var(--#{$prefix}tab-content-padding-y) * -1) calc(var(--#{$prefix}tab-content-padding-x) * -1); // stylelint-disable-line function-disallowed-list
```

Comment:
OK for me but makes the whole thing a bit harder/longer to read/understand...

Reply:
:warning: `calc(var(--#{$prefix}tab-content-padding-y) * -1) ` -> `calc(-1 * var(--#{$prefix}tab-content-padding-y))` (check all the PR)